prql-compiler/tests/integration/snapshots/integration__fmt@group_sort.prql.snap

```diff
@@ -4,9 +4,9 @@ expression: "from tracks\nderive d = album_id + 1\ngroup d (\n    aggregate {\n
 input_file: prql-compiler/tests/integration/queries/group_sort.prql
 ---
 from tracks
-derive d = album_id + 1
-group d (aggregate {n1 = (track_id | sum)})
+derive d=album_id + 1
```

Comment:
Is this easy to read?

```elm
derive d=album_id + 1
```

I feel that it would be difficult to understand which of the following calculation order is used.

```elm
derive (d=album_id) + 1
```

```elm
derive d=(album_id + 1)
```

Reply:
No, I agree, that seems bad.

What are some options for that?

```
derive d=(album_id + 1)
```

...might be reasonable — i.e. compound expressions go in paretheses?

Reply:
We could have very tight expressions; i.e. 

```
derive d=album_id+1
```

...but I don't think that'll scale well?

Reply:
I prefer the followings:

```elm
derive d=(album_id + 1)
```


```elm
derive d = (album_id + 1)
```

(I prefer to have spaces before and after the equals first of all.)

Reply:
On second though, I'd vote for:

```
derive d = album_id + 1
```

It looks a bit confusing for join, but it's not that bad:

```
join s = salaries (==employee_id)
join m = manager e.manager_id == m.id
```